### PR TITLE
Alpine image and cmake for rclient

### PR DESCRIPTION
**Pull request by beeender**

### CODE REVIEW

**Review by yihong0618 — COMMENTED** [review]
one more space here?

**Comment by beeender** [comment]
> one more space here?

fixed

**Review by yihong0618 — APPROVED** [review]
LGTM
passed on my local env.

File: src/rclient/bin/rclient.sh [hunk 1]
```diff
@@ -1,10 +1,11 @@
-#!/bin/bash
+#!/bin/sh
 # ------------------------------------------------------------------------------
 #
 # Copyright (c) 2016-Present Pivotal Software, Inc
 #
 # ------------------------------------------------------------------------------
 cd /clientdir
 
+export LD_LIBRARY_PATH="/clientdir:$LD_LIBRARY_PATH"
```
Comment (Sasasu) [inline src/rclient/bin/rclient.sh:9]: for `librcall.so`.

```
$ ldd rclient
  libm.so.6 => /usr/lib/libm.so.6 (0x00007f4be683c000)                                            [0/18]
  libreadline.so.8 => /usr/lib/libreadline.so.8 (0x00007f4be7247000)
  libpcre2-8.so.0 => /usr/lib/libpcre2-8.so.0 (0x00007f4be679f000)
  liblzma.so.5 => /usr/lib/liblzma.so.5 (0x00007f4be721d000)
  libbz2.so.1.0 => /usr/lib/libbz2.so.1.0 (0x00007f4be678c000)
  libz.so.1 => /usr/lib/libz.so.1 (0x00007f4be6772000)
  libtirpc.so.3 => /usr/lib/libtirpc.so.3 (0x00007f4be6743000)
  libicuuc.so.71 => /usr/lib/libicuuc.so.71 (0x00007f4be6544000)
  libicui18n.so.71 => /usr/lib/libicui18n.so.71 (0x00007f4be6212000)
  libgomp.so.1 => /usr/lib/libgomp.so.1 (0x00007f4be61c9000)
  libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x00007f4be61a9000)
  /lib64/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007f4be734a000)
  libgfortran.so.5 => /usr/lib/libgfortran.so.5 (0x00007f4be5ece000)
  libncursesw.so.6 => /usr/lib/libncursesw.so.6 (0x00007f4be5e5a000)
  libgssapi_krb5.so.2 => /usr/lib/libgssapi_krb5.so.2 (0x00007f4be5e05000)
  libicudata.so.71 => /usr/lib/libicudata.so.71 (0x00007f4be4100000)
  libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x00007f4be3ec9000)
  libquadmath.so.0 => /usr/lib/../lib/libquadmath.so.0 (0x00007f4be3e80000)
  libkrb5.so.3 => /usr/lib/libkrb5.so.3 (0x00007f4be3da7000)
  libk5crypto.so.3 => /usr/lib/libk5crypto.so.3 (0x00007f4be3d77000)
  libcom_err.so.2 => /usr/lib/libcom_err.so.2 (0x00007f4be7211000)
  libkrb5support.so.0 => /usr/lib/libkrb5support.so.0 (0x00007f4be3d68000)
  libkeyutils.so.1 => /usr/lib/libkeyutils.so.1 (0x00007f4be3d61000)
  libresolv.so.2 => /usr/lib/libresolv.so.2 (0x00007f4be3d4e000)
```
and looks our released image does not build from `dockerfiles/Dockerfile.r`

docker inspect pivotaldata/plcontainer_r_shared:devel | jq  ".[].ContainerConfig.Env"

---

plc should use a normal container image, I support this change.

**Review by Sasasu — APPROVED** [review]

**Comment by Sasasu** [comment]
and why `librcall.so` is not a static link...

**Comment by beeender** [comment]
> and why librcall.so is not a static link...

```
❯ ldd rclient/librcall.so
        linux-vdso.so.1 (0x00007fff32ffd000)
        libc.musl-x86_64.so.1 => not found
```
The so built from cmake should be OK.

**Pull request merged**